src/components/Icons/Arrow.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { memo, FC } from 'react';
+import styled from 'styled-components';
+
+type Direction = 'left' | 'right' | 'up' | 'down';
+
+type ArrowProps = {
+	direction?: Direction;
+};
+
+const Arrow: FC<ArrowProps> = memo(({ direction = 'left' }) => (
+	// @ts-ignore
```

Comment:
not sure why its complaining about `direction` being a string... I clearly put the `Direction` type